src/format/mod.rs

```diff
@@ -280,6 +280,63 @@ enum InternalInternal {
     Nanosecond9NoDot,
 }
 
+/// Type for specifying the format of UTC offsets.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub struct OffsetFormat {
+    /// See `OffsetPrecision`.
+    pub precision: OffsetPrecision,
+    /// Seperator between hours, minutes and seconds.
```

Comment:
One more instance of `sepe`

Reply:
Thank you for being thorough, I thought I had them all.